spec/kibana_spec.rb

```diff
@@ -1,24 +1,9 @@
 # encoding: utf-8
 require 'spec_helper'
 
-kibana_version = '4.4.1'
```

Comment:
Let's keep the var in the tests. Now that the package is installed from an apt repo, we can use the `package` resource type to and check that it `should be_installed`. We can also make sure it's installed at 4.5-or-greater, which is an appropriate check starting now.